Initium-Odp/src/com/universeprojects/miniup/server/services/CombatService.java

```diff
@@ -64,4 +65,26 @@ public boolean isInCombat(CachedEntity character)
 		else
 			return false;
 	}
-}
+	
+	/**
+	 * Removes all items from the specified character for which the strength requirement
+	 * is no longer met (i.e. a buff is applied or expires).
+	 */
+	public void updateEquips(CachedEntity character){
+		CachedDatastoreService ds = db.getDB();
+		for(String slot:ODPDBAccess.EQUIPMENT_SLOTS)
+		{
+			Key itemKey = (Key)character.getProperty("equipment"+slot);
+			CachedEntity item = db.getEntity(itemKey);
```

Comment:
It's better to do a batch get instead of hitting the server separately for each piece.